package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "paella-engage-ui",
-  "version": "4.0.10",
+  "version": "4.0.11",
```

Comment:
Jim, does the 4.0.10 have to also be updated in the npm-shrinkwrap.json or can that stay 4.0.10 until the shrinkwrap versions are changed?